src/Surfnet/StepupSelfService/SelfServiceBundle/Controller/RemoteVettingController.php

```diff
@@ -210,46 +202,28 @@ public function remoteVetMatchAction(Request $request, $processId)
         /** @var SamlToken $samlToken */
         $samlToken = $this->container->get('security.token_storage')->getToken();
 
-        $attributes = AttributeListDto::fromAttributeSet($samlToken->getAttribute(SamlToken::ATTRIBUTE_SET));
+        $localAttributes = AttributeListDto::fromAttributeSet($samlToken->getAttribute(SamlToken::ATTRIBUTE_SET));
```

Comment:
Consider naming this variable identity-provider-attributes as I later refer to them with that moniker when they get encrypted.

Reply:
I've added a commit and renamed the 'identity-provider-attributes' to 'local-attributes'. Because local and remote are both IdP's.